sdcm/nemesis.py

```diff
@@ -862,6 +862,8 @@ def disrupt_major_compaction(self):
         self.target_node.run_nodetool("compact")
 
     def disrupt_nodetool_refresh(self, big_sstable: bool = False):
+        if self.cluster.params.get("cluster_backend") in ['aws-siren', 'gce-siren']:
```

Comment:
@Orenef11 please look into changing this to look at the cluster/node version instead, so we won't forget about it. 
https://github.com/scylladb/scylla-cluster-tests/blob/f2ced40e7b3f1f899a12f698091754309c318608/sdcm/cluster.py#L4561

and 

https://github.com/scylladb/scylla-cluster-tests/blob/f2ced40e7b3f1f899a12f698091754309c318608/sdcm/cluster.py#L4575

```python
# skip test if it's older enterprise version
if self.cluster.is_enterprise and parse_version(self.cluster.scylla_version) < parse_version("2021.1"):
```

Reply:
@fruch
We do not want skip on these nemesis for all of Siren's JOBs?

Reply:
Done

Reply:
This change not worked for Sirenada
```
14:54:04  < t:2021-02-24 12:54:03,918 f:nemesis.py      l:2894 c:sdcm.nemesis         p:ERROR > sdcm.nemesis.ScyllaCloudLimitedChaosMonkey: Unhandled exception in method <function ScyllaCloudLimitedChaosMonkey.disrupt at 0x7f1ee49d4700> < t:2021-02-24 12:54:03,918 f:nemesis.py      l:2894 c:sdcm.nemesis         p:ERROR > sdcm.nemesis.ScyllaCloudLimitedChaosMonkey: Unhandled exception in method <function ScyllaCloudLimitedChaosMonkey.disrupt at 0x7f1ee49d4700>
14:54:04  < t:2021-02-24 12:54:03,918 f:nemesis.py      l:2894 c:sdcm.nemesis         p:ERROR > Traceback (most recent call last):
14:54:04  < t:2021-02-24 12:54:03,918 f:nemesis.py      l:2894 c:sdcm.nemesis         p:ERROR >   File "/jenkins/slave/workspace/longevity-tests/cloud-longevity-small-data-set-1h-gcp/scylla-cluster-tests/sdcm/nemesis.py", line 2884, in wrapper
14:54:04  < t:2021-02-24 12:54:03,918 f:nemesis.py      l:2894 c:sdcm.nemesis         p:ERROR >     result = method(*args, **kwargs)
14:54:04  < t:2021-02-24 12:54:03,918 f:nemesis.py      l:2894 c:sdcm.nemesis         p:ERROR >   File "/jenkins/slave/workspace/longevity-tests/cloud-longevity-small-data-set-1h-gcp/scylla-cluster-tests/sdcm/nemesis.py", line 3312, in disrupt
14:54:04  < t:2021-02-24 12:54:03,918 f:nemesis.py      l:2894 c:sdcm.nemesis         p:ERROR >     self.call_random_disrupt_method(disrupt_methods=['disrupt_nodetool_cleanup',
14:54:04  < t:2021-02-24 12:54:03,918 f:nemesis.py      l:2894 c:sdcm.nemesis         p:ERROR >   File "/jenkins/slave/workspace/longevity-tests/cloud-longevity-small-data-set-1h-gcp/scylla-cluster-tests/sdcm/nemesis.py", line 1061, in call_random_disrupt_method
14:54:04  < t:2021-02-24 12:54:03,918 f:nemesis.py      l:2894 c:sdcm.nemesis         p:ERROR >     self.execute_disrupt_method(disrupt_method)
14:54:04  < t:2021-02-24 12:54:03,918 f:nemesis.py      l:2894 c:sdcm.nemesis         p:ERROR >   File "/jenkins/slave/workspace/longevity-tests/cloud-longevity-small-data-set-1h-gcp/scylla-cluster-tests/sdcm/nemesis.py", line 1068, in execute_disrupt_method
14:54:04  < t:2021-02-24 12:54:03,918 f:nemesis.py      l:2894 c:sdcm.nemesis         p:ERROR >     disrupt_method()
14:54:04  < t:2021-02-24 12:54:03,918 f:nemesis.py      l:2894 c:sdcm.nemesis         p:ERROR >   File "/jenkins/slave/workspace/longevity-tests/cloud-longevity-small-data-set-1h-gcp/scylla-cluster-tests/sdcm/nemesis.py", line 867, in disrupt_nodetool_refresh
14:54:04  < t:2021-02-24 12:54:03,918 f:nemesis.py      l:2894 c:sdcm.nemesis         p:ERROR >     if self.cluster.is_enterprise and parse_version(self.cluster.scylla_version) < parse_version("2021.1"):
14:54:04  < t:2021-02-24 12:54:03,918 f:nemesis.py      l:2894 c:sdcm.nemesis         p:ERROR > AttributeError: 'ScyllaCloudCluster' object has no attribute 'is_enterprise'
```
https://jenkins.scylladb.com/view/siren/job/longevity-tests/job/cloud-longevity-small-data-set-1h-gcp/56/consoleFull

Reply:
I revert this change

Reply:
seem that `is_enterprise` is part of  `BaseMonitorSet` or part of `BaseNode`, please implment it also on `BaseScyllaCluster`, and then it might work.